documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -63,32 +74,64 @@ spec:
 ----
 endif::product[]
 
+[NOTE]
+====
+Automatic topic creation requires the `replication.factor` and `partitions` properties to be set for at least the `default` topic creation group.
```

Comment:
See above, I think we should not name this "automatic topic creation". Would be interesting to see how it's described in the Kafka KIP.

Reply:
Topic auto-creation = topic automatic creation = automatic topic creation, doesn't it?  I'm not sure what is your objection. Do you think that using the same terminology to refer to Kafka Connect topic creation and broker topic creation is too confusing? 

In the KIP we see the following phrase: _"Kafka Connect should instead be able to create the topics automatically for source connectors ..."_ Elsewhere there's a reference to _"creation of new topics during runtime"_ and _"topic creation during runtime."_

We could instead refer to _"runtime topic creation"_. WDYT?

Reply:
Perhaps it needs qualifying? "Kafka Connect automatic topic creation"? Or "Automatic topic creation driven by Kafka Connect"?

> Do you think that using the same terminology to refer to Kafka Connect topic creation and broker topic creation is too confusing?

Yes, that's my concern. But perhaps it's not a problem if things become clear from the context.